src/Microsoft.Health.Dicom.Functions/Configs/ReindexConfiguration.cs

```diff
@@ -6,13 +6,14 @@
 using System;
 using System.ComponentModel.DataAnnotations;
 
-namespace Microsoft.Health.Dicom.Functions.Indexing.Configuration
+namespace Microsoft.Health.Dicom.Functions.Configs
 {
     /// <summary>
     /// Represents the configuration for a "reindex" function.
     /// </summary>
     public class ReindexConfiguration
```

Comment:
ReIndexJobConfiguration?

Reply:
I will change this to `ReindexOperationConfiguration`, since we decided to call it `ReindexOperation` instead of `ReindexJob` in design doc.